src/Commands/LightningCommands.php

```diff
@@ -77,7 +73,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
    *   The semantic version number.
    */
   public static function toSemanticVersion($drupal_version) {
-    preg_match('/^8\.x-(\d+).(\d)(\d+)(-.+)?$/', $drupal_version, $matches);
+    preg_match('/^9\.x-(\d+).(\d)(\d+)(-.+)?$/', $drupal_version, $matches);
```

Comment:
Let's modify this a bit -- change it to `/^[89]\.x` (and so forth), to allow either 8.x or 9.x here. That should fix the test which is currently failing.

Reply:
Forgot to change that part. Thanks!